ceremonies/src/tests.rs

```diff
@@ -1540,30 +1900,32 @@ fn grow_population_works() {
         for pair in participants.iter() {
             let proof = match EncointerCeremonies::participant_reputation(
                 (cid, cindex - 1),
-                get_accountid(&pair),
+                account_id(&pair),
             ) {
-                Reputation::VerifiedUnlinked => Some(prove_attendance(
-                    get_accountid(&pair),
-                    cid,
-                    cindex - 1,
-                    &pair,
-                )),
+                Reputation::VerifiedUnlinked => {
+                    Some(prove_attendance(account_id(&pair), cid, cindex - 1, &pair))
+                }
                 _ => None,
             };
             EncointerCeremonies::register_participant(
-                Origin::signed(get_accountid(&pair)),
+                Origin::signed(account_id(&pair)),
                 cid,
                 proof,
-            );
+            )
+            .unwrap();
         }
         run_to_next_phase();
         // ASSIGNING
         assert_eq!(EncointerCeremonies::meetup_count((cid, cindex)), 2);
         let meetup4_1 = EncointerCeremonies::meetup_registry((cid, cindex), 1);
         let meetup4_2 = EncointerCeremonies::meetup_registry((cid, cindex), 2);
 
-        // whitepaper III-B Rule 3: no more than 1/4 participants without reputation
-        assert!(meetup4_1.len() + meetup4_2.len() <= 13);
+        // whitepaper III-B Rule 3: no more than 1/3 participants without reputation
```

Comment:
@brenzi what do you think?

Reply:
I agree:

ceremony1:
6B + 14N => 9 assigned, 1 meetup
ceremony2:
6B+ 3R + 11N => 13 assigned, 2 meetups
ceremony3:
6B+ 7R + 7N => 19 assigned, 2 meetups